doc/users/event_handling.rst

```diff
@@ -403,100 +383,77 @@ background that the mouse is over::
 
     plt.show()
 
-
 .. _object-picking:
 
 Object picking
 ==============
 
-You can enable picking by setting the ``picker`` property of an
-:class:`~matplotlib.artist.Artist` (e.g., a matplotlib
-:class:`~matplotlib.lines.Line2D`, :class:`~matplotlib.text.Text`,
-:class:`~matplotlib.patches.Patch`, :class:`~matplotlib.patches.Polygon`,
-:class:`~matplotlib.image.AxesImage`, etc...)
+You can enable picking by setting the ``picker`` property of an `.Artist` (such
+as `.Line2D`, `.Text`, `.Patch`, `.Polygon`, `.AxesImage`, etc.)
 
-There are a variety of meanings of the ``picker`` property:
+The ``picker`` property can be set using various types:
 
     ``None``
-	picking is disabled for this artist (default)
-
+        Picking is disabled for this artist (default).
     ``boolean``
-	if True then picking will be enabled and the artist will fire a
-	pick event if the mouse event is over the artist
-
-    ``float``
-	if picker is a number it is interpreted as an epsilon tolerance in
-	points and the artist will fire off an event if its data is
-	within epsilon of the mouse event.  For some artists like lines
-	and patch collections, the artist may provide additional data to
-	the pick event that is generated, e.g., the indices of the data
-	within epsilon of the pick event.
-
-    ``function``
-	if picker is callable, it is a user supplied function which
-	determines whether the artist is hit by the mouse event.  The
-	signature is ``hit, props = picker(artist, mouseevent)`` to
-	determine the hit test.  If the mouse event is over the artist,
-	return ``hit=True`` and props is a dictionary of properties you
-	want added to the :class:`~matplotlib.backend_bases.PickEvent`
-	attributes
-
+        If True, then picking will be enabled and the artist will fire a
+        pick event if the mouse event is over the artist.
+    ``callable``
+        If picker is a callable, it is a user supplied function which
+        determines whether the artist is hit by the mouse event.  The
+        signature is ``hit, props = picker(artist, mouseevent)`` to
+        determine the hit test.  If the mouse event is over the artist,
+        return ``hit = True``; ``props`` is a dictionary of properties that
+        become additional attributes on the `.PickEvent`.
+
+The artist's ``pickradius`` property can additionally be set to a tolerance
+value (in points; there are 72 points per inch) that determines how far the
```

Comment:
```suggestion
value in points (there are 72 points per inch) that determines how far the
```
IMHO the unit is first-class information that should not be parenthesed.

If you think that's too ambiguous, use e.g. `tolerance value in units of points`